UnlGadt1: Uncaught exception ghc-internal:GHC.Internal.Exception.ErrorCall:

boom

HasCallStack backtrace:
  collectBacktraces, called at libraries/ghc-internal/src/GHC/Internal/Exception.hs:91:13 in ghc-internal:GHC.Internal.Exception
  toExceptionWithBacktrace, called at libraries/ghc-internal/src/GHC/Internal/Exception.hs:118:5 in ghc-internal:GHC.Internal.Exception
  error, called at UnlGadt1.hs:20:13 in main:Main

